airflow/contrib/sensors/DataLake_PrefixFile_Sensor.py

```diff
@@ -0,0 +1,54 @@
+from airflow.contrib.hooks.azure_data_lake_hook import AzureDataLakeHook
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+
+
+# File detector by prefix sensor for azure data lake storage
+
+class DataLakePrefixSensor(BaseSensorOperator):
+
+    """
+    Interacts with Azure Data Lake:
+
+    Client ID and client secret should be in user and password parameters.
+    Tenant and account name should be extra field as
+    {"tenant": "<TENANT>", "account_name": "ACCOUNT_NAME"}.
+
+    :param azure_data_lake_conn_id: Reference to the Azure Data Lake connection
+
+    DataLake_path : directory of the file
+
+    prefix : file name
```

Comment:
Can you update the docstring for the parameters to match the Sphinx style used. The `azure_data_lake_conn_id` above is a good example or other examples in operators and sensors.